task/pins/pin-mini.svg

```diff
@@ -0,0 +1,21 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="22" height="31" viewBox="0 0 22 31">
+    <defs>
+        <ellipse id="5g4mnr6yib" cx="10" cy="29" rx="4" ry="2"/>
+        <filter id="or8n4twdga" width="262.5%" height="425%" x="-81.2%" y="-137.5%" filterUnits="objectBoundingBox">
+            <feOffset dy="1" in="SourceAlpha" result="shadowOffsetOuter1"/>
+            <feGaussianBlur in="shadowOffsetOuter1" result="shadowBlurOuter1" stdDeviation="2"/>
+            <feColorMatrix in="shadowBlurOuter1" result="shadowMatrixOuter1" values="0 0 0 0 0.0470588235 0 0 0 0 0.0470588235 0 0 0 0 0.0549019608 0 0 0 0.2 0"/>
+            <feOffset in="SourceAlpha" result="shadowOffsetOuter2"/>
+            <feGaussianBlur in="shadowOffsetOuter2" result="shadowBlurOuter2" stdDeviation="1"/>
+            <feColorMatrix in="shadowBlurOuter2" result="shadowMatrixOuter2" values="0 0 0 0 0.0470588235 0 0 0 0 0.0470588235 0 0 0 0 0.0549019608 0 0 0 0.12 0"/>
+            <feMerge>
+                <feMergeNode in="shadowMatrixOuter1"/>
+                <feMergeNode in="shadowMatrixOuter2"/>
+            </feMerge>
+        </filter>
+    </defs>
+    <g fill="none" fill-rule="evenodd" transform="translate(1 1)">
+        <!-- <use fill="#000" filter="url(#or8n4twdga)" xlink:href="#5g4mnr6yib"/> -->
```

Comment:
Do you know if the `filter` is what causes spritezero to ignore the transparency ?  
Could a [`<radialGradient>`](https://developer.mozilla.org/fr/docs/Web/SVG/Element/radialGradient) used in the `fill` attribute be supported instead ?

Reply:
Yep I planned to give it a go. Will keep you informed :)

Reply:
No luck for now, as soon as there is some alpha level involved, whatever the syntax, the sprite generator tool renders a black shape. I would advise to postpone the resolution of this detail for now, as it may involve diving into spritezero implemetation :|

Reply:
I found this ref, as spritezero uses Mapnik for SVG manipulation: https://github.com/mapnik/mapnik/wiki/SVG-support
I was apparently wrong about all alpha levels, if I give it a try with an opacity on the main path it's correctly rendered in the sprite:
![Capture d’écran de 2020-10-15 18-00-08](https://user-images.githubusercontent.com/243653/96155545-55e69400-0f10-11eb-8310-1d559402fcf3.png)

I'll try other ways with the gradient.